HotSpot/MapSearchViewController.m

```diff
@@ -0,0 +1,84 @@
+//
+//  MapSearchViewController.m
+//  HotSpot
+//
+//  Created by aodemuyi on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "MapSearchViewController.h"
+#import "MapKit/MapKit.h"
+#import "SearchCell.h"
+
+@interface MapSearchViewController () <UITableViewDataSource, UITableViewDelegate>
```

Comment:
I think we should rename this file. Maybe just SearchViewController would do for now, but something a little more specific would be ideal, like ParkingSearchViewController. can you come up with a suitable name?